electron/legendary_utils/library.ts

```diff
@@ -209,10 +209,7 @@ class Library {
       dlcItemList,
       releaseInfo,
       categories,
-      customAttributes : {
-        CloudSaveFolder,
-        FolderName
-      }
+      customAttributes
```

Comment:
Important bugfix, heroic does not start, because unreal assets do not have a cloudsavefolder

Reply:
Yeah, we noticed this when we released 1.5.8. It was fixed on 1.5.9